packages/create-keystone-app/example-projects/starter/initial-data.js

```diff
@@ -0,0 +1,51 @@
+// This script will check if the User table is empty and if
+// required seed the table with an initial Admin user.
+const randomString = () =>
+  Math.random()
+    .toString(36)
+    .substring(2, 6) +
+  Math.random()
+    .toString(36)
+    .substring(2, 6);
```

Comment:
`Math.random()` is not very good.  How about something like this?

```
const crypto = require('crypto');
const randomString = () => crypto.randomBytes(6).hexSlice();
```

Reply:
This is not intended to be a secure password by any measure. I thought about using `12345678` and honestly that might even be better because I want people to change it.

In the end I went random just incase someone accidentally deploys with this.

Reply:
Now having defended my decision I realise `crypto` is native node and the code is shorter... so why not have it be better.